src/main/kotlin/app/revanced/patches/youtube/misc/video/quality/patch/RememberVideoQualityPatch.kt

```diff
@@ -58,6 +60,51 @@ class RememberVideoQualityPatch : BytecodePatch(
             )
         )
 
+        // This is bloated and needs a better way to add a single key that holds an array of values.
```

Comment:
That doesn't make sense. You can use the following to instantiate an array of StringResource which you can then use in the patch to avoid duplicate code. I've used the same approach in another patch already:

arrayOf("144").withIndex().map { i, k => StringResource("entry_$i", k) }

Reply:
Ah i see, I was referring to reducing duplicate code in the patch.